### PR TITLE
[TypeScript][Bot-Solutions] Add Extensions test folder 

### PR DESCRIPTION
**Pull request by DiegoCardozo94**
### Purpose
*What is the context of this pull request? Why is it being done?*

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

File: sdk/typescript/libraries/botbuilder-solutions/test/extensions/dateTimeEx.test.js [hunk 1]
```diff
@@ -7,15 +7,15 @@ require("dayjs/locale/de");
...
 const dayjs = require("dayjs");
 const { join } = require("path");
 const i18next = require("i18next").default;
 const i18nextNodeFsBackend = require("i18next-node-fs-backend");
-const { DateTimeTestData } = require("./helpers/dateTimeTestData");
-const { Locales } = require(join("..", "lib", "localesUtils"));
-const { DateTimeEx } = require(join("..", "lib", "extensions", "dateTimeEx"));
+const { DateTimeTestData } = require("../helpers/dateTimeTestData");
+const { Locales } = require("../../lib/localesUtils");
+const { DateTimeEx } = require(join("..", "..", "lib", "extensions", "dateTimeEx"));
```
Comment (Batta32) [inline sdk/typescript/libraries/botbuilder-solutions/test/extensions/dateTimeEx.test.js:18]:
```suggestion
const { DateTimeTestData } = require(join("..", "helpers", "dateTimeTestData"));
const { Locales } = require(join("..", "..", "lib", "localesUtils"));
const { DateTimeEx } = require(join("..", "..", "lib", "extensions", "dateTimeEx"));
```

File: sdk/typescript/libraries/botbuilder-solutions/test/extensions/dateTimeEx.test.js [hunk 2]
```diff
@@ -7,15 +7,15 @@ require("dayjs/locale/de");
 require("dayjs/locale/es");
 require("dayjs/locale/it");
 require("dayjs/locale/fr");
-require("../lib/resources/customizeLocale/zh");
+require("../../lib/resources/customizeLocale/zh");
```
Comment (Batta32) [inline sdk/typescript/libraries/botbuilder-solutions/test/extensions/dateTimeEx.test.js:10]:
```suggestion
require(join("..", "..", "lib", "resources", "customizeLocale", "zh"));
```

File: sdk/typescript/libraries/botbuilder-solutions/test/extensions/luisRecognizeEx.test.js [hunk 3]
```diff
@@ -5,9 +5,9 @@
 
 const { strictEqual } = require("assert");
 const { join } = require("path");
-const { SkillLuis } = require(join(__dirname, "helpers", "skillLuis"));
-const { LuisRecognizerEx } = require(join("..", "lib", "extensions", "luisRecognizerEx"));
-const { SentimentType } = require(join("..", "lib", "models", "sentimentType"));
+const { SkillLuis } = require("../helpers/skillLuis");
```
Comment (Batta32) [inline sdk/typescript/libraries/botbuilder-solutions/test/extensions/luisRecognizeEx.test.js:8]:
```suggestion
const { SkillLuis } = require(join("..", "helpers", "skillLuis"));
```

File: sdk/typescript/libraries/botbuilder-solutions/test/extensions/dateTimeEx.test.js [hunk 4]
```diff
@@ -7,15 +7,15 @@ require("dayjs/locale/de");
...
-require("../lib/resources/customizeLocale/zh");
 const { strictEqual } = require("assert");
 const dayjs = require("dayjs");
 const { join } = require("path");
 const i18next = require("i18next").default;
 const i18nextNodeFsBackend = require("i18next-node-fs-backend");
-const { DateTimeTestData } = require("./helpers/dateTimeTestData");
-const { Locales } = require(join("..", "lib", "localesUtils"));
-const { DateTimeEx } = require(join("..", "lib", "extensions", "dateTimeEx"));
+const {customLocaleZh} = require(join("..", "..", "lib", "resources", "customizeLocale", "zh"));
```
Comment (Batta32) [inline sdk/typescript/libraries/botbuilder-solutions/test/extensions/dateTimeEx.test.js:15]: I don't think that this is correctly used... you should leave as it was before but only including the `join` inside the `require`

Comment (DiegoCardozo94) [reply sdk/typescript/libraries/botbuilder-solutions/test/extensions/dateTimeEx.test.js:15]: I have one question about this, i cannot use the join before declare it, so what could be the correct way of implement this?

Comment (Batta32) [reply sdk/typescript/libraries/botbuilder-solutions/test/extensions/dateTimeEx.test.js:15]: Is it no possible to do `require(join("..", "..", "lib", "resources", "customizeLocale", "zh"));`?

**Pull request merged**